planner/core/casetest/testdata/integration_suite_out.json

```diff
@@ -1602,14 +1607,14 @@
         "SQL": "explain format = 'brief' select /*+ hash_join_build(t) */ * from t where exists (select /*+ SEMI_JOIN_REWRITE() */ 1 from t t1 join t t2 where t1.a = t2.a and t1.a = t.a)",
```

Comment:
The hint doesn't take effect because of the semi join rewrite. It will be handled in the next PR.